src/abhishekSJun20/IntersectionOfArray.java

```diff
@@ -0,0 +1,69 @@
+package abhishekSJun20;
+/*Program 2: Find intersection(common elements) of two arrays.
+Input: arr1 -> {10.45, 14.0, 18.35, 88.88, 54.10, 18.35}
+          arr2 -> {17.20, 13.30, 10.45, 18.35, 84.33, 13.30}
+Output: 10.45, 18.35
+Method Signature : double[] getIntersectionOfArray(double[] arr1, double[] arr2)*/
+
+
+public class IntersectionOfArray  {
+    int count = 0;
+	void findIntersectionOfArray(double[] arr1, double[] arr2) {
+		int sizeOfIntersectionArray = arr1.length + arr2.length;
+		double[] arr3 = new double[sizeOfIntersectionArray];
+		// compared first Array arr1 with arr2 and formed new array arr3 with common elements of arr1 and arr2
+		for (int outerindex = 0; outerindex < arr1.length; outerindex++) {
+			for (int innerindex = 0; innerindex < arr2.length; innerindex++) {
+				if (arr1[outerindex] == arr2[innerindex]) {
+					arr3[count] = arr1[outerindex];
+					count++;
+				}
+			}
+		}
+		//from arr3 find duplicate elements and replace duplicates with zero
```

Comment:
Add logic accordingly, no need to replace duplicate with zeroes, what if in given input 0 is there. So instead of it, store common elements in one array, that logic is more appropriate.